curp/src/server/cmd_board.rs

```diff
@@ -25,7 +22,7 @@ pub(super) struct CommandBoard<C: Command> {
     /// Store all notifiers for conf change results
     conf_notifier: HashMap<ProposeId, Event>,
     /// Store all conf change results
-    pub(super) conf_buffer: IndexMap<ProposeId, Result<bool, ConfChangeError>>,
+    pub(super) conf_buffer: IndexSet<ProposeId>,
```

Comment:
Why should the conf_buffer be an IndexSet?

Reply:
Because the result of conf change is already merged into `handle_propose_conf_change`, we don't need pass this result from `Filter` to `propose_conf_change`